kivy_ios/toolchain.py

```diff
@@ -1171,6 +1178,14 @@ def _pip(args):
     shprint(pip_cmd, *args, _env=pip_env)
 
 
+def _hostpython_pip(args):
+    ctx = Context()
+    pip_path = join(ctx.dist_dir, 'hostpython3', 'bin', 'pip3')
+    logger.error("Executing pip for hostpython with: {}".format(args))
```

Comment:
Why `logger.error` rather than `logger.info`?

Reply:
I assume I copy-pasted from here (I already did a git blame and guess Who's the author 🤦  ):
https://github.com/kivy/kivy-ios/blob/6b245b102dbac76428f9807f25056c9e1bd53ce6/kivy_ios/toolchain.py#L1169

Fixed both. Will push within the CI fixes.

Reply:
Ahah don't blame the contributor, blame the reviewer :sweat_smile: 
Thanks for the fix and yeah I also love git blaming :smile: